### ISSUE TITLE
Request for wiki account:  for Real

### ISSUE THREAD

**Issue by python-coding-404** [webmin/webmin#1734]
For real dude for Real give me a wiki account

**Comment by unknown10777** [comment]
![image](https://user-images.githubusercontent.com/83267603/188650722-3c05d66c-a53d-40f9-ae91-f68e29589cd8.png)
I'm a litttttle confusing about your issues.

**Comment by python-coding-404** [comment]
Please just give me a wiki account. - Love python-coding-404

**Comment by swelljoe** [comment]
Sorry for the slow reply. I'll need your email address to create the account (so it can send you a one-time confirmation link to set your password). If you don't want to post it here, you can PM me on our forums at forums.virtualmin.com or email me at swelljoe at gmail.

**Comment by python-coding-404** [comment]
@swelljoe  toomanyalt.acc.alt2@gmail.com

**Comment by python-coding-404** [comment]
@swelljoe toomanyalt.acc.alt@gmail.com

**Comment by swelljoe** [comment]
Cool, check your email for a confirmation and password reset link!

Thanks for volunteering to help out with the wiki! I'm sorry I haven't been very responsive. I think my notifications for this repo are going to the wrong email address or something...gonna have to look into it.

Please let me know if you have any questions of whatever about the wiki. It's a mess, as it's been very lightly maintained (I got frustrated with the relentless onslaught of spam before I turned off account creation and my brain kinda shutdown and lost all enthusiasm for wikis in general, but the Webmin wiki specifically).

**Comment by python-coding-404** [comment]
where is the email i have been waiting for two days

**Comment by python-coding-404** [comment]
I know that that the server are far far far far across continents but I checked on the email but no luck
I even check the spam
where is the email?

**Comment by swelljoe** [comment]
Dang, I don't know. Let me check the log to see if I can find what happened. It was supposed to send you an email to set your own password (and has always worked in the past).

**Comment by swelljoe** [comment]
Apparently we're blocked by Google.

```
Dec  3 01:09:33 swelljoe postfix/smtp[3841936]: 00BB51C1C231: to=<toomanyalt.acc.alt@gmail.com>, relay=none, delay=348867, delays=348717/0.06/150/0, dsn=4.4.1, status=deferred (connect to alt2.gmail-smtp-in.l.google.com[142.250.147.26]:25: Connection timed out)
```

I don't know why that'd be so, and I have historically received mail from it at my GMail account. I don't know how to proceed now. MediaWiki won't let me reset your password, nor will it let me delete and recreate an account. MediaWiki is so annoying...

**Comment by swelljoe** [comment]
I just dug up the email and sent it to you from my personal account. Let me know if you're able to get in now.

Apologies for the inconvenience. We'll be migrated the wiki into a static-site generator (probably Hugo) at some point, and it'll get a github that allows PRs to be made against it in the future. But, we've gotta get the Webmin site done, first.

**Comment by swelljoe** [comment]
Well, that didn't work either.

```
550 5.1.1 The email account that you tried to reach does not exist. Please try double-checking the recipient's email address for typos or unnecessary spaces. Learn more at https://support.google.com/mail/?p=NoSuchUser pk25-20020a0568704c1900b00131ccdbbc9bsor2882274oab.0 - gsmtp 
```

**Comment by swelljoe** [comment]
And, I guess that explains why MediaWiki couldn't send you a password. I'm sending to `toomanyalt.acc.alt@gmail.com`.

**Comment by python-coding-404** [comment]
Oh yeah that happens 
That why i gave my alt acc even my main has this issue
like i have this account i can login but it happened when i use main acc and try to send a msg to that alt
only some or service taht ai sign up before it happend can still send
looks like i gonna have to send my outlook account

**Comment by python-coding-404** [comment]
express-emu@outlook.com

**Comment by python-coding-404** [comment]
Nothing to preview

**Comment by python-coding-404** [comment]
also yes wikpedia can accses the account that you cant send to

**Comment by python-coding-404** [comment]
Why every developer just goes poof gone for issues that was made last year
help

**Comment by swelljoe** [comment]
I'm sorry for the delay. @iliaross has been working on converting the docs from a wiki to a Hugo site, that will be checked into git (probably already is, though it's not yet done). We'll be taking the current docs offline and replacing them with a static site generated with Hugo quite soon...so, changes, additions, etc. can be made with PRs here at github.

**Comment by iliajie** [comment]
Yes, quite soon! I will do my best wrapping it all up within next few weeks!

**Comment by python-coding-404** [comment]
still nothing

**Comment by swelljoe** [comment]
@iliaross can I publish what we have? It includes all the existing docs, right? I know you want to do updates, but we need to get off the wiki so other folks can work on it.

**Comment by iliaross** [comment]
> @iliaross can I publish what we have? It includes all the existing docs, right?

You can see what we have using the following link. At the moment SSH Server is the last:

https://webmin.dev/docs/modules/ssh-server

There is more work to do!

**Comment by swelljoe** [comment]
That doesn't solve the problem we're trying to address. What github repo is this in?

**Comment by iliajie** [comment]
> That doesn't solve the problem we're trying to address. What github repo is this in?

It's here [webmin/webmin.com/content/docs/Modules](https://github.com/webmin/webmin.com/tree/main/content/docs/Modules).

**Comment by swelljoe** [comment]
Oh, so we are still missing a lot of stuff (all the tutorials and such).

But, that does mean @python-coding-404 can make PRs to suggest improvements.

**Comment by python-coding-404** [comment]
Found the wiki
